### ISSUE TITLE
[Jasper/Pytorch] Do I need to modify the default parameters of train.sh in the case of a single GPU?

### ISSUE THREAD

**Issue by zhijinghe** [NVIDIA/DeepLearningExamples#925]
Currently, 10 epochs are trained, and the loss decreases slowly and remains at about 120. Do I need to modify other parameters except the GPU number during single GPU training?

**Comment by alancucki** [comment]
Hi @zhijinghe ,

sorry for a laggy reply.

The default recipe is for `NUM_GPUS=8 BS=64`. With a single GPU this requires `NUM_GPUS=1 BS=512` to maintain global BS. To fit this all in GPU memory, adjust gradient accumulation steps, e.g. `GRAD_ACCUMULATION_STEPS=16` or a similar value, depending on the actual memory usage.

Note that with mixed precision training (`AMP=true`) you can fit larger batches, and decrease the number of accumulation steps.
